API/NoDocs/Aspire.StackExchange.Redis/StackExchangeRedisSettings.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Aspire.StackExchange.Redis'
+
+using System.Runtime.CompilerServices;
+
+namespace Aspire.StackExchange.Redis;
+
+public sealed class StackExchangeRedisSettings
```

Comment:
@sebastienros was 100% correct in the meeting. These `Settings` classes can't use the `IOptions` pattern because the `Settings` objects influence what gets put into the DI container. `Settings.Tracing = false` means "don't put the Tracing services into DI". Since these Settings objects need to be evaluated before the DI service provider is built, they can't be `IOptions`, which is why we choose the name `Settings`.

See:

https://github.com/dotnet/aspire/blob/e2ba459e1505c82219efef201d086481c1f59df2/src/Components/README.md?plain=1#L38-L40